src/rard/research/views/fragment.py

```diff
@@ -1010,3 +1010,136 @@ def fetch_fragments(request):
+    else:
+        original_fragment = get_object_or_404(Fragment, pk=pk)
+    original_original_texts = original_fragment.original_texts.all()
+
+    new_original_texts = []
+
+    for i in range(len(original_original_texts)):
+        # Create a dictionary to hold the values for the new OriginalText object
+        new_original_text_data = {}
+
+        # Iterate over the fields of the OriginalText model
+        for field in original_original_texts[i]._meta.fields:
+            # Exclude the ID field
+            if field.name in ["id", "created", "modified"]:
+                continue
+
+            field_value = getattr(original_original_texts[i], field.name)
+
+            # For ForeignKey fields, copy the related object
+            if field.is_relation and field_value is not None:
+                field_value = field_value
+
+            new_original_text_data[field.name] = field_value
+
+        # Create a new OT object with the copied values
+        new_original_text = OriginalText.objects.create(**new_original_text_data)
+        new_original_texts.append(new_original_text)
+
+        # Recreate References for new OT
+        for reference in original_original_texts[i].references.all():
+            Reference.objects.create(
+                editor=reference.editor,
+                reference_position=reference.reference_position,
+                original_text=new_original_text,
+            )
+
+        # Copy relationships to Concordances, Ap Crit & Translations
+        model_details = {
+            Concordance: {
+                "filter_name": "original_text",
+                "filter_value": original_original_texts[i],
+                "ot_fieldname": "original_text",
+            },
+            ApparatusCriticusItem: {
+                "filter_name": "object_id",
+                "filter_value": original_original_texts[i].pk,
+                "ot_fieldname": "parent",
+            },
+            Translation: {
+                "filter_name": "original_text",
+                "filter_value": original_original_texts[i],
+                "ot_fieldname": "original_text",
+            },
+        }
+
+        for model_class in model_details.keys():
+            model_info = model_details[model_class]
+            filter_name = model_info["filter_name"]
+            filter_value = model_info["filter_value"]
+            ot_fieldname = model_info["ot_fieldname"]
+
+            new_model_data = {}
+            for original_item in model_class.objects.filter(
+                **{filter_name: filter_value}
+            ):
+                for field in original_item._meta.fields:
+                    # Exclude some fields
+                    if field.name in [
+                        "id",
+                        "created",
+                        "modified",
+                        ot_fieldname,
+                        "content_type",
+                    ]:
```

Comment:
You can also add object_id to this list as it and content type are inferred from parent.
```suggestion
                        "object_id",
                    ]:
```